imports/izar_protocol_v3.leo

```diff
@@ -0,0 +1,270 @@
+program izar_protocol_v3.aleo {
+    mapping izar_keeper: bool => Keepers20;
+    mapping izar_digest: bool => scalar;
+    // 0: current_epoch, 1: current_keeper_num;
+    mapping izar_config: u8 => u32;
+    mapping nonces: Nonce => bool;
+
+    struct Nonce {
+        chain_id: u32,
+        nonce: u128,
+    }
+
+    inline contract_owner() -> address {
+        return aleo1yu6ssnehj4plgznatenfut3hkrfus26kqd226k99wfx79jqmsuxqfernns;
+    }
+
+    inline unvalid_contract() -> address {
+        return aleo1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq3ljyzc;
+    }
+
+    /* config setup */
+    transition set_epoch (epoch: u32) {
+        assert_eq(self.caller, contract_owner());
+        return then finalize (epoch);
+    }
+
+    finalize set_epoch (epoch: u32) {
+        assert(epoch > Mapping::get_or_use(izar_config, 0u8, 0u32));
+        Mapping::set(izar_config, 0u8, epoch);
```

Comment:
It seems the `epoch` is never used?

Reply:
Yes, keep here for governance purpose. We should remove it when it is never used. Fixed.